crates/ruff_dev/Cargo.toml

```diff
@@ -42,6 +42,9 @@ strum = { workspace = true }
 strum_macros = { workspace = true }
 tempfile = "3.6.0"
 toml = { workspace = true, features = ["parse"] }
+tracing = { workspace = true }
```

Comment:
Can we remove `log` or is it used by other dev tools?

Reply:
yes!